w3-blob.md

```diff
@@ -118,127 +118,57 @@ AcceptOk -- site --> Site
 
 Shown Invocation example illustrates Alice requesting to add 2MiB blob to her space.
 
-```js
+> ℹ️ Note: we use `// "/": "bafy..` comments to denote CID of the parent object.
+
+```jsonc
 { // "/": "bafy..add"
-  "cmd": "/space/content/add/blob",
-  "sub": "did:key:zAlice",
   "iss": "did:key:zAlice",
-  "aud": "did:web:web3.storage",
-  "args": {
-    "blob": {
-      // multihash of the blob as byte array
-      "digest": { "/": { "bytes": "mEi...sfKg" } },
-      // size of the blob in bytes
-      "size": 2_097_152,
+  "aud": "did:web:up.storacha.network",
+  "att": [{
+    "can": "space/blob/add",
+    "with": "did:key:zAlice",
```

Comment:
`did:key:zAlice` is Alice's key, right? I see the did is also used in a lot of places where a space did is expected, such as the `with` here. Should the with read something like `"with": "did:key:zAliceSpace"`?

If that's the case, the same happens in many other `with`s through the document.

Reply:
ACK, yes I'll clarify.